release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,74 @@ vf:fulfills     a owl:ObjectProperty ;
 #        rdfs:range      xsd:boolean ;
 #        rdfs:comment    "The commitment or process is complete or not." .
 
-# isSubstitutable "Defines if any resources of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc."
+vf:isSubstitutable a    owl:DatatypeProperty ;
+        rdfs:label      "is substitutable" ;
+        rdfs:domain     vf:ResourceType ;
+        rdfs:range      xsd:boolean ;
+        rdfs:comment    "Defines if any resource of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc." .
+
+#vf:unit a               owl:ObjectProperty ;
+#        rdfs:label      "unit" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:range      qudt:unit ;
+#        rdfs:comment    "The unit expected for this resource type as a default." .
+
+vf:definedQuantity a    owl:ObjectProperty ;
+        rdfs:label      "defined quantity" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The amount and unit defined for this recipe flow." .
+
+# TODO: need to define this, something like currency, inventory, work, service; may be other aspects too.
+#vf:resourceCategory a    owl:ObjectProperty ;
+#        rdfs:label      "category" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:comment    "Defines the category of behavior expected for the type of resource." .
+
+vf:developedBy a        owl:ObjectProperty ;
+        rdfs:label      "developed by" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      foaf:Agent ;
+        rdfs:comment    "The agent who developed this recipe." .
+
+vf:variantOf a           owl:ObjectProperty ;
+        rdfs:label      "variant of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is a variation of another recipe." .
+
+vf:successorOf a        owl:ObjectProperty ;
+        rdfs:label      "successor of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is an updated version of another recipe." .
+
+vf:actsOn a             owl:ObjectProperty ;
```

Comment:
Can we simply use something `vf:affectedClassifiedAs`. Somehow we don't have such property to reference taxonomy of a resource affected by flow, we need something like that for Intent and Commitment and I think we can just use it also with RecipeFlow.

Reply:
I guess that depends on the resolution of if we need a RecipeResource thing rather than just a classification.  I like "affects" better than "acts on" -- but we already used that.... and again might depend on if we have a RecipeResource thing.

Reply:
Changed name to affected.

Reply:
:-1: to similar to `vf:affects` and if it references classification of the resource `vf:affectedClassifiedAs` seems much more clear. Actually we shouldn't handle it in Recipes PR since we also seem to need at `vf:Intent` and `vf:Commitment` (*produce*, *consume*, *use*) to reference classification of a resource affected by that flow.

Reply:
>-1 to similar to vf:affects and if it references classification of the resource vf:affectedClassifiedAs seems much more clear.

Well right now it doesn't reference a classification, it references something on the knowledge layer, and we have finally gotten clear on the difference.  I realize you don't agree that should exist. :)

>Actually we shouldn't handle it in Recipes PR since we also seem to need at least vf:Intent and vf:Commitment (produce, consume, use) to reference classification of a resource affected by that flow.

But the recipe won't hang together without it.  We might need vf:Intent and vf:Commitment to reference this knowledge layer class, good point - but right now they can reference the classification.  We can add that before doing this PR if you like.  But we'll still need the name.